pycbc/events/stat.py

```diff
@@ -963,6 +963,13 @@ def __init__(self, sngl_ranking, files=None, ifos=None, **kwargs):
         # This will be used to keep track of the template number being used
         self.curr_tnum = None
 
+        # Applies a constant offset to all statistic values in a given instance.
+        # This can be used to e.g. change relative rankings between different
+        # event types. Default is zero offset.
+        self.stat_correction = float(
+            self.kwargs.get("statistic_correction", 0)
+        )
+
         # Go through the keywords and add class information as needed:
         if self.kwargs["sensitive_volume"]:
             # Add network sensitivity beckmark
```

Comment:
we may as well fix this typo as well 
```suggestion
            # Add network sensitivity benchmark
```